robotoff/scheduler/__init__.py

```diff
@@ -202,11 +203,14 @@ def _refresh_elasticsearch():
 # this job does no use database
 def _update_data():
     """Refreshes the PO product dump and updates the Elasticsearch index data."""
-
-    _download_product_dataset()
-    # Elasticsearch is dependent on the availability of the PO product dump, i.e.
-    # it it called after the download product dataset call.
-    _refresh_elasticsearch()
+    try:
+        _download_product_dataset()
+        # Elasticsearch is dependent on the availability of the product dump from Product Opener
+        # (main Open Food Facts backend)
+        # it it called after the download product dataset call.
+        _refresh_elasticsearch()
+    except requests.exceptions.ConnectionError:
```

Comment:
I would prefer using `requests.exceptions.RequestException` to catch any type of request errors instead of just a `ConnectionError`